### PR TITLE
#20700 Fix ecr vulnerabilities thecodingmachine/gotenberg

### PR DESCRIPTION
**Pull request by robbydrmwn**

### CODE REVIEW

**Review by daniel-jeremia — APPROVED** [review]

File: charts/datasaur/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
```
Comment (tonoman3g) [inline charts/datasaur/Chart.yaml:18]: is it okay if we bump the minor version @daniel-jeremia ?

Comment (daniel-jeremia) [reply charts/datasaur/Chart.yaml:18]: usually we use bump patch version for updates like this, it wouldn't change anything, just a convention @tonoman3g @robbydrmwn

Comment (robbydrmwn) [reply charts/datasaur/Chart.yaml:18]: ohh.. should I change it to 2.1.1 then? @daniel-jeremia

Comment (tonoman3g) [reply charts/datasaur/Chart.yaml:18]: ganti ajaa @robbydrmwn

Comment (robbydrmwn) [reply charts/datasaur/Chart.yaml:18]: updated ya @tonoman3g @daniel-jeremia

**Review by daniel-jeremia — APPROVED** [review]

**Pull request merged**